clients/bazel-module-registry/src/main/kotlin/LocalBazelModuleRegistryService.kt

```diff
@@ -33,7 +33,7 @@ const val METADATA_JSON = "metadata.json"
 const val SOURCE_JSON = "source.json"
 
 /**
- * A Bazel registry for the workspace [directory] which is located on the local file system. It is configured as [url]
```

Comment:
Hmm, clicking `directory` and `url` actually works for me to jump to the constructor arguments. So nothing is broken here for me (using IntelliJ 2025.1.1.1).